portfolio/src/main/webapp/script.js

```diff
@@ -13,12 +13,13 @@
 // limitations under the License.
 
 
-
-
 /**
- * Adds a random greeting to the page.
+ * Changes image and updates caption in gallery section
+ * dir = 1 goes to next image
+ * dir = -1 goes to previous image
  */
 function changePhoto(dir) {
+
```

Comment:
nit: remove extra line here

Reply:
I have fixed this issue in the commit "Remove space after changePhoto function"